src/logic/utils.ts

```diff
@@ -41,13 +56,18 @@ export function getAverageWinTurn(player: Player) {
     return average.toFixed(1);
 }
 
+/**
+ * Used to define what highlights a newly qualified player.
+ * @param player 
+ * @returns True / False
```

Comment:
:D this is too literal of a description, instead of something like: "A boolean representing if the player is considered "newly qualified".

Reply:
fixed.